Tests/Subset.py

```diff
@@ -310,6 +310,32 @@ def test_update_subset_dynamic_public(self):
             '{{ [{}].[EUR], [{}].[USD] }})'.format(self.dimension_name, self.dimension_name))
 
         self.assertNotEqual(self.dynamic_subset, subset)
+        
+    def test_make_static_private(self):
+        private = True
+        subset = self.tm1.dimensions.subsets.get(
+            subset_name=self.dynamic_subset.name,
+            dimension_name=self.dynamic_subset.dimension_name,
+            private=private)
+        self.assertTrue(subset.is_dynamic)
+        self.tm1.dimensions.hierarchies.subsets.make_static(
+            subset_name=self.dynamic_subset.name,
+            dimension_name=self.dynamic_subset.dimension_name,
+            private=private)
+        self.assertTrue(subset.is_static)
```

Comment:
You must retrieve the subset again before you do the check:

``` python
        subset = self.tm1.dimensions.subsets.get(
            subset_name=self.dynamic_subset.name,
            dimension_name=self.dynamic_subset.dimension_name,
            private=private)
        self.assertTrue(subset.is_static)
```

Reply:
sorry, taken care.